docs/multinode/config_rpm.md

```diff
@@ -0,0 +1,37 @@
+# RHEL with MicroShift RPMs
+Log into the `hypervisor host` and follow the instructions described in the
+[Getting Started with MicroShift](../getting_started.md) document to create the
+`microshift-pri` and `microshift-sec` virtual machines for running primary and
+secondary instances.
```

Comment:
Should the reader follow all of the instructions, or just the parts for creating the VMs?